app/controllers/users_controller.rb

```diff
@@ -21,6 +21,10 @@ def create
     end
   end
 
+  def contact_us
+    
```

Comment:
Add route as a comment?
Also maybe this should go to the pages controller as it doesn't have much to do with users